README.md

```diff
@@ -0,0 +1,93 @@
+# tlsdialer
```

Comment:
It would be nice to have godoc badge:
https://github.com/tarantool/go-tarantool/blob/f02579a563687a53600f9988b7eba7ebced293c8/README.md?plain=1#L5

And coverage:
https://github.com/tarantool/go-tarantool/blob/f02579a563687a53600f9988b7eba7ebced293c8/README.md?plain=1#L7